tests/v2/validation/configmaps/configmaps_test.go

```diff
@@ -0,0 +1,107 @@
+package configmaps
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+  "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+  
+)
+
+type SteveAPITestSuite struct {
```

Comment:
nit: can we rename this to ConfigmapTestSuite so its more appropriate for the tests.